stubs/ldap3/ldap3/core/connection.pyi

```diff
@@ -59,7 +60,7 @@ class Connection:
     post_send_search: Any
     def __init__(
         self,
-        server: Server | str,
+        server: Server | str | Iterable[Server] | ServerPool,
```

Comment:
Could you add a comment that that not all arbitrary iterables/sequences are supported? Alternatively, we could list the allowed sequences as a union:

Somewhere above (needs appropriate imports):

```python
_ServerSequence = set[Server] | list[Server] | tuple[Server, ...] | Generator[Server, None, None] | dict_keys[Server, Any]
```

Here:

```python
        server: Server | str | _ServerSequence | ServerPool,
```

Reply:
Done. Added the explicit iterable typtes defined in the package using your example. Thank you for the help here.